dataplane/dplanerc/interface.go

```diff
@@ -737,29 +790,50 @@ func (ni *Reconciler) handleAddrUpdate(ctx context.Context, au *netlink.AddrUpda
 			sub.GetOrCreateIpv6().DeleteAddress(ip)
 			gnmiclient.BatchDelete(sb, ocpath.Root().Interface(intf.name).Subinterface(intf.subintf).Ipv6().Address(ip).State())
 		}
-		_, err := ni.fwdClient.TableEntryRemove(ctx, &fwdpb.TableEntryRemoveRequest{
-			ContextId: &fwdpb.ContextId{Id: ni.contextID},
-			TableId:   &fwdpb.TableId{ObjectId: &fwdpb.ObjectId{Id: cpusink.IP2MeTable}},
-			EntryDesc: entry.Build(),
-		})
-		if err != nil {
-			log.Warningf("failed to remove route: %v", err)
-			return
+		// For an aggregate interface, add an IP2ME route for all of it's members.
```

Comment:
Is this part of the logic duplicate with the above?

Reply:
somewhat, do it's calling different func with a different request type. i didn't see a good to dedupe it